mesonbuild/compilers/c.py

```diff
@@ -1284,9 +1288,13 @@ def __init__(self, exelist, version, compiler_type, is_cross, exe_wrapper=None,
 
     def get_options(self):
         opts = CCompiler.get_options(self)
+        c_stds = ['c89', 'c99', 'c11']
+        g_stds = ['gnu89', 'gnu99', 'gnu11']
+        if version_compare(self.version, '>=8.0.0'):
+            c_stds += ['c17','c18']
```

Comment:
__[Flake8]__ [E231] missing whitespace after ',' ([link](https://sider.review/gh/repos/19784232/pulls/5062#issue-3829496))

<sub>posted by Sider</sub>

Reply:
I wasn't even consisten: space separation of elements in some list literals but not others. I'll push a squashed commit to fix that. Just pushed it, now I'll to squash it.